Dockerfile

```diff
@@ -13,6 +13,10 @@ COPY dockerbuild/rsyslog.conf /etc/rsyslog.conf
 RUN mkdir -p /opt/ssl
 COPY dockerbuild/logentries.all.crt /opt/ssl/logentries.all.crt
 
+# get s3-expand
+RUN curl https://raw.githubusercontent.com/silinternational/s3-expand/master/s3-expand -o /usr/local/bin/s3-expand
```

Comment:
Would it be worth pointing at a specific version of s3-expand, in case we even need to make a backwards-incompatible change to s3-expand?